scripts/eurostat/gdp/eurostat_gdp.tmcf

```diff
@@ -10,7 +10,7 @@ unit: dcs:Euro
 
 Node: E:eurostat_gdp->E2
 typeOf: dcs:StatVarObservation
-variableMeasured: dcid:Amount_EconomicActivity_GrossDomesticProduction_AsAFractionOfCount_Person
+variableMeasured: dcid:Amount_EconomicActivity_GrossDomesticProduction_AsAFractionOfCount_Person_Nominal
```

Comment:
Also change to Amount_EconomicActivity_GrossDomesticProduction_Nominal_PerCapita here

Reply:
Done.